CONTRIBUTING.md

```diff
@@ -26,9 +26,6 @@ The file is gitignored, so feel free to keep it around in your local copy of the
 HUB=gcr.io/xyz # consider localhost:5000
 TAG=ambient
 
-# Optional: build rust ztunnel
```

Comment:
this is still useful for development

Reply:
Ah i see, I was using Mac instructions which has this env var set in the build cmd.

Reply:
restored